buildSrc/src/main/kotlin/elastic-otel.sign-and-publish-conventions.gradle.kts

```diff
@@ -1,3 +1,5 @@
+import gradle.kotlin.dsl.accessors._2a2fda20f7c0d5ad930aaa9c8e47b6e1.jar
```

Comment:
[minor] is this import needed here ?

Reply:
Good point, this looks wrong to me. I've replaced it by a lookup using the name as string.